packages/fantasion-backend/fantasion_eshop/models.py

```diff
@@ -265,9 +265,36 @@ class Meta:
     @property
```

Comment:
Když to dělá dotazy do db, tak už by to potom neměla být property, skrývá se tím komplexita.